tests/treewalker/c_sharp_spec.lua

```diff
@@ -41,4 +41,12 @@ describe("In a C Sharp file", function()
     assert.same(first_block, lines.get_lines(30, 48))
     h.assert_cursor_at(7, 5)
   end)
+
+  it("swaparound behavior works", function()
+    vim.fn.cursor(52, 31) -- (|node1, node2)
```

Comment:
Can we remove all these -- (|node1, node2) comments where they're not accurate? Otherwise this is ready for merge!